skills.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  
+  <head>
+    <title>Skills</title>
+    <link href="css/styles.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Rubik:300,400,500" rel="stylesheet" type="text/css">
+  </head>
+  
+  <body>
+    
+    <a href="index.html" class="home" alt="home" title="home">←</a>
+
+    <header>
+      
+      <h1>
+        I CAN DO THESE THINGS
```

Comment:
@catherine-rosenwald try to wrap your text within `<p>` tags. for example
`<p> ...text ...</p>`